sourced/ml/mining/models/stdlib.py

```diff
@@ -0,0 +1,63 @@
+from modelforge import merge_strings, Model, register_model, split_strings
+from sourced.ml.core.models.license import DEFAULT_LICENSE
+
+
+@register_model
+class StandardLibraries(Model):
+    """
+    List of standard library contents for each Babelfish language.
+    """
+
+    NAME = "stdlib"
+    VENDOR = "source{d}"
+    DESCRIPTION = (
+        "The list of standard library contents for C++, C#, Go, Java, JavaSript, PHP, "
+        "Python and Ruby. Language version is not taken into account, data was collected "
```

Comment:
```suggestion
        "Python and Ruby. The language versions are not taken into account. The data is collected "
```

Reply:
actually since we keep it and other stuff `infos`, I'll rephrase this a bit, tell me what you think after